ACPs/75_consensus_proofs/75-consensus-proofs.md

```diff
@@ -0,0 +1,87 @@
+```text
+ACP: 75
+Title: Consensus Proofs
+Author(s): Joshua Kim
+Discussions-To: https://github.com/avalanche-foundation/ACPs/discussions/76
+Status: Proposed
+Track: Standards
+```
+
+## Abstract
+
+Introduces support for a proof of a block’s acceptance in consensus.
+
+## Motivation
+
+Subnets use the ProposerVM to determine the block producer for the next block in the chain through soft-leader election. ProposerVM determines the block producer schedule from a randomly shuffled validator set for that subnet at each block height.  Each block proposed by an elected block producer contains the height of the P-Chain parent block it should be verified at, so that other verifying validators can verify that the block was produced at a valid time.
+
+One key invariant is that nodes cannot verify a block that was produced at a P-Chain height that they have not heard of yet. Verifiers assume that a block height that is beyond the P-Chain height that they have accepted is invalid because that block does not exist yet from the verifier’s perspective.
+
+If many nodes disagree about the current tip of the P-Chain, it can lead to a liveness failure where a subnet is not able to produce any blocks because nodes might have copies of the P-Chain that are out-of-sync with each other. In practice, this almost never happens because nodes produce blocks with a P-Chain height in the past, using a few blocks as a buffer since it’s likely that most nodes would have accepted an old block. This however, relies on an assumption that validators are constantly making progress in consensus to prevent the subnet from potentially stalling. This leaves an open concern where the P-Chain stalling on a node would prevent it from verifying any blocks, leading to a subnet potentially unable to produce blocks if many validators stalled at different heights due to a P-Chain outage.
+
+If ProposerVM blocks were produced with a proof of the P-Chain block to be verified against instead of just its height, verifiers could continue making progress by requesting any block they had not accepted yet from another peer and verifying their corresponding acceptance proofs. If a block’s proof is valid, the blocks can be executed in-order locally to finally verify the proposed subnet block.
```

Comment:
Do we want to more explicitly callout that this could also be used by non-validating nodes to accept blocks instead of running consensus (which would mean many less connections + les load on validators)?

Reply:
Yeah I can add this as a section.